pkg/recutil/reconcile.go

```diff
@@ -18,10 +19,12 @@ import (
 )
 
 const (
-	EventStart    = "Start"
-	EventNotFound = "NotFound"
-	EventError    = "Error"
-	EventComplete = "Complete"
+	EventRequestStart = "ReconcileRequestStart"
```

Comment:
I prefixed all of these with `Reconcile`, as while it's arguably redundant given that we're in a reconcile package, it felt too ambiguous when viewing events on an object to see just 'Start' or 'Complete'.